### DISCUSSION TITLE
What does EVAL do?

### DISCUSSION THREAD

**Discussion by jpgpng** [modula3/cm3#1080] [General]
Why we need EVAL? Isn't `FINALLY Process.Wait(p)` is enough? Why it must be `FINALLY EVAL Process.Wait(p)`?

https://modula3.github.io/cm3/help/gen_html/libm3/src/os/Common/Process.i3.html

**Comment by mtbc** [comment]
It's good to be explicit about throwing away return values so that we don't accidentally ignore one that we should be using.

**Comment by jpgpng** [comment]
It's really hard to find on the reference. Sorry.

https://modula3.github.io/cm3/reference/complete/html/2_3_3Eval.html
